tests/contrib/pytest/test_pytest.py

```diff
@@ -290,3 +292,28 @@ def test_service(ddspan):
         file_name = os.path.basename(py_file.strpath)
         rec = self.subprocess_run("--ddtrace", file_name)
         assert 0 == rec.ret
+
+    def test_dd_origin_tag_on_every_span(self):
```

Comment:
would it make sense to use the test agent on this?

Reply:
What do you mean?

Reply:
```suggestion
from tests.utils import snapshot

    @snapshot()
    def test_dd_origin_tag_on_every_span(self):
```

although, it looks like it won't "just work", maybe a future improvement for all of these test cases.

Reply:
I think snapshotting should probably be added for these test cases in general, I'm just concerned that for the dd_origin tests (which only add origin tags and nothing else to each trace, which is small to begin with) snapshotting may not be so beneficial.